### PR TITLE
RAINCATCH-1315 - show dialog on invalid redirect

### PR DESCRIPTION
**Pull request by wtrocki**
## Motivation

When ?url= is missing and application is not pointing to the separate url we should show dialog to inform users.

### CODE REVIEW

**Comment by wtrocki** [comment]
Review and verification required.

File: packages/angularjs-auth-passport/lib/webAuthService.js [hunk 1]
```diff
@@ -104,4 +115,16 @@ WebAuthService.prototype.logout = function() {
   });
 }
 
+
+/**
+ * Private - show dialog when cloudUrl is missing
+ */
+WebAuthService.prototype.showAlertDialog = function() {
```
Comment (paolobueno) [inline packages/angularjs-auth-passport/lib/webAuthService.js:122]: Maybe give this method a more descriptive name or generic callout, such as `showMissingCloudUrlDialog()` or `showAlertDialog(new Error('Application is missing cloud server url. Please see documentation for proper configuration'))`

**Review by paolobueno — APPROVED** [review]
Verified locally, optional comment!

File: packages/angularjs-auth-passport/lib/webAuthService.js [hunk 2]
```diff
@@ -104,4 +115,16 @@ WebAuthService.prototype.logout = function() {
...
+  return self.dialog.show(self.dialog.alert({
```
Comment (JameelB) [inline packages/angularjs-auth-passport/lib/webAuthService.js:123]: uses self but self is not defined

Comment (paolobueno) [reply packages/angularjs-auth-passport/lib/webAuthService.js:123]: +1, didn't catch this, I wonder why eslint didn't too though

Comment (paolobueno) [reply packages/angularjs-auth-passport/lib/webAuthService.js:123]: I see, looks like we don't have eslint setup for some of the packages, I'll add a `npm test` script that at least runs it for the ones that are missing in a separate PR

Comment (wtrocki) [reply packages/angularjs-auth-passport/lib/webAuthService.js:123]: Not sure how this was working on my local machine :D

**Pull request merged**